text/0039-sdk-performance-issue-creation.md

```diff
@@ -0,0 +1,67 @@
+* Start Date: 2022-11-28
+* RFC Type: decision
+* RFC PR: [#33](https://github.com/getsentry/rfcs/pull/33)
+* RFC Status: draft
+* RFC Driver: [Philipp Hofmann](https://github.com/philipphofmann)
+
+# Summary
+
+This RFC aims to clarify which types of performance issues SDKs shall create.
+
+# Motivation
+
+On June 21, 2022, we decided with [DACI](https://www.notion.so/sentry/Performance-Issue-Creation-POC-e521772ebccb482b83b08f4f8a3db2cb) to create performance issues in Ingest. While implementing the file I/O on the main thread performance issue for Android, the question arose as to why SDKs don't report easy-to-detect and matured performance issues. 
+
+# Options Considered
+
+## Option 1: SDKs report easy to detect and matured performance issues
+
+We still need to clarify what easy to detect and matured precisely means, which is not the goal of this document. For example, file I/O on the main thread could be a candidate.
+
+To clarify the threshold and configuration for performance issues, an experimental feature phase can help to get feedback.
+
+### Pros
+
+1. SDKs can capture a stack trace which will help with actionability.
+2. No running transaction required.
+3. Sentry can tie together transaction and error far more easily since both objects exist at the point of time the performance issue will be created.
```

Comment:
I don't think this is true, since errors and transactions have different quotas its possible (and common) that errors exist without a transaction

Reply:
yeah, we literally say at 2) that no running transaction is required :) or we're misunderstanding the phrasing here

Reply:
🤔 Right I think its the "far more easily" here I have issue with since its actually not that easy until Dynamic Sampling can do Error Biasing